src/main/java/com/dev/cinema/controllers/MovieSessionMapper.java

```diff
@@ -0,0 +1,49 @@
+package com.dev.cinema.controllers;
```

Comment:
```suggestion
package com.dev.cinema.service.mappers;
```

let's move all mappers to `service/mappers`